src/components/screens/monitor/delegates/index.js

```diff
@@ -13,32 +14,70 @@ import withLocalSort from '../../../../utils/withLocalSort';
 import withResizeValues from '../../../../utils/withResizeValues';
 import NotAvailable from '../notAvailable';
 
-const defaultUrlSearchParams = { tab: 'active' };
+const defaultUrlSearchParams = { search: '' };
 const delegatesKey = 'delegates';
+const standByDelegatesKey = 'standByDelegates';
 
 const mapStateToProps = ({ blocks: { latestBlocks } }) => ({
   latestBlocks,
 });
 
+const transformResponse = (response, oldData, urlSearchParams) => (
+  urlSearchParams.offset
+    ? [...oldData, ...response.filter(
+      delegate => !oldData.find(({ username }) => username === delegate.username),
```

Comment:
IndexOf is alooooooot faster than this iteraton. please read this
https://wsvincent.com/javascript-remove-duplicates-array/

Reply:
we can keep this based on what we discuss